vision/emergent_object/emergent_object_processing.py

```diff
@@ -0,0 +1,161 @@
+"""Picks the most likely emergent object out of the pool of candidates"""
+
+from nptyping import NDArray, Shape, Float64
+
+import numpy as np
+import geopy.distance
+
```

Comment:
`geopy` is not in the standard library, so it will need to be added to the poetry environment with `poetry add geopy`.